db/migrate/20160410033943_add_default_value_to_wiki_attribute.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultValueToWikiAttribute < ActiveRecord::Migration
+  def change
+    change_column :wikis, :private, :boolean, :default => false
+  end
+end
```

Comment:
@bjblock - I realized that my original issue with the Wikis not changing to private had to do with the attribute being stored as nil right from the get go, so I added this migration to set the default value to false. I haven't been able to get the view to work yet so I haven't been able to try it out just yet but I did have a question. Do I need to pass the `checkbox` form within the view `:true` to make the attribute change when the box is checked?